vcr/patch.py

```diff
@@ -406,23 +413,24 @@ def reset_patchers():
             yield mock.patch.object(cpool.HTTPConnectionPool, 'ConnectionCls', _cpoolHTTPConnection)
             yield mock.patch.object(cpool.HTTPSConnectionPool, 'ConnectionCls', _cpoolHTTPSConnection)
 
+    # FIXME this is not right probably
     try:
-        import botocore.vendored.requests.packages.urllib3.connectionpool as cpool
+        import botocore.awsrequest as cpool
```

Comment:
same here, old and new imports, plus that FIXME?

Reply:
Fix me is there because I wasn't really sure it works this way.
And I need more details on how this works. 
However it behaved correctly with latest boto3 versions